jenkins_init.sh

```diff
@@ -93,7 +93,7 @@ check_docker_compose() {
 generate_ssh_keys() {
   local tutorial_path=$1
   echo "generating new ssh keys"
-  sh $tutorial_path/keygen.sh $tutorial_path
+  bash $tutorial_path/keygen.sh $tutorial_path
```

Comment:
Why not call the script directly?
Are there any challenges or considerations that I may not be aware of?